packages/frontend/src/hooks/dashboard/useDashboards.tsx

```diff
@@ -13,11 +13,11 @@ export const getDashboards = async (projectUuid: string, chartId?: string) => {
     });
 };
 
-export const useDashboards = (projectUuid: string) => {
+export const useDashboards = (projectUuid: string, chartId?: string) => {
     const setErrorResponse = useQueryError();
     return useQuery<DashboardBasicDetails[], ApiError>({
         queryKey: ['dashboards', projectUuid],
-        queryFn: () => getDashboards(projectUuid || ''),
+        queryFn: () => getDashboards(projectUuid, chartId || ''),
         enabled: projectUuid !== undefined,
         onError: (result) => setErrorResponse(result),
     });
```

Comment:
We should have a different function/hook for this since they are for different use cases. One is to fetch all dashboards the other is to fetch all the dashboards that have that chart.
Example of what can happen with this code:
1. user deletes a chart
2. useQuery fetch the dashboards that have a chart and caches the results
3. user goes to dashboards page 
4. useQuery returns the cached results ( which is incorrect )

Note for the new hook: when we update create/update/delete dashboards we should invalidate the cache of the new hook.